client/js/components/GameView/Card.vue

```diff
@@ -35,11 +35,11 @@ export default {
   name: 'Card',
   props: {
     suit: {
-      type: Number,
+      type: [Number, String],
       required: true,
     },
     rank: {
-      type: Number,
+      type: [Number, String],
```

Comment:
Why can these be strings?

Reply:
Okay I see it's for the 'back' case. Is that necessary? The current implementation doesn't use the `Card` component for card backs. Curious what you and @seriouslysean both think about this. It's more generic this way, which is perhaps more flexible. 

Maybe it makes more sense to have a boolean `back` prop and to create a computed prop to return the path to the image? Then the question becomes what values if any to give for the `suit` and `rank` props. We could either
1. Pass the suit and rank anyway even though they are ignored when rendering the component b/c `back = true`. This arguably is less secure because the user isn't supposed to know which cards are in place given that these cards are displayed as card backs, but if you're already able to read the non-rendered values of props in the component hierarchy then you can already see that stuff in the store, so I think this might be the best/cleanest
2. Make those props optional and/or accept null values and set them to null for card back images

What do y'all think?

Reply:
Agreed that 'back' feels a bit awkward here, but I do like the flexibility and consistency in styling with visible cards that adding this to `Card.vue` provides.

'1' Definitely seems like the most clean approach, although as the game grows I fear that people would realize over that the rank and suit are visible on face-down cards. Null definitely seems like a potential option as well! 

What if we used a bogus numerical value like `99`?